### DISCUSSION TITLE
Portrait/Edits Support

### DISCUSSION THREAD

**Discussion by undaunt** [steilerDev/icloud-photos-sync#537] [Q&A]
Hello,

I just wanted to confirm, but it appears to be the case that portrait mode photos are somehow properly supported with this tool, unlike icloud-photos-downloader. I thought this was a limitation of pyicloud only downloading master images but perhaps not.

In any case, is there a way to customize whether the original, edited, or both files are downloaded?

Thank you.

**Comment by steilerDev** [comment]
For portrait mode photos, there is an "original" that does not include the blur (or any edits) and an "edited" file, that includes the currently selected blur.

Since the primary reason for this tool was a backup scenario, currently original versions and their edit (if any) are always downloaded. I would like to rework some of the configurations to allow more flexibility - but I have not yet come around doing that.
